cassandra/cluster.py

```diff
@@ -3592,6 +3600,10 @@ def _try_connect(self, host):
                     peers_query, timeout=self._timeout)
 
             shared_results = (peers_result, local_result)
+
+            if len(shared_results) < 2:
+                raise DriverException("Database not sending proper response for SELECT on system.local or system.peers table")
```

Comment:
nit:
`Database not sending proper response for SELECT on system.local or system.peers table`
to
`Did not receive complete response for SELECT on system.local or system.peers table`